cmd/kured/main.go

```diff
@@ -483,9 +495,24 @@ func deleteNodeAnnotation(client *kubernetes.Clientset, nodeID, key string) {
 }
 
 func rebootAsRequired(nodeID string, rebootCommand []string, sentinelCommand []string, window *timewindow.TimeWindow, TTL time.Duration, releaseDelay time.Duration) {
-	config, err := rest.InClusterConfig()
-	if err != nil {
-		log.Fatal(err)
+
+	log.Infof("In rebootAsRequired")
+	var config *rest.Config
+	var err error
+
+	if runtime.GOOS == windows {
```

Comment:
If we implement interface with windows/linux k8s servers as types, then we could have a method for getting the config based on those. It would look simpler and simpler to test.

Reply:
I'll look into building an interface to for windows/linux behaviors so we can remove the GOOS if/else statements in the code.
I wanted to cleanup some other parts of this PR first (like building the container images with make targets) but that is done.